telegram/ext/conversationhandler.py

```diff
@@ -168,7 +168,6 @@ class ConversationHandler(Handler):
         ValueError
 
     """
-
```

Comment:
This one's probably left from the previous changes. Can you remove it? Just looks confusing in the git log